calibrations/mbd/submit_mbdcalibs.sh

```
@@ -0,0 +1,50 @@
+for fname in ${SUBMITDIR}/runlists/?????.list
+do
+  echo ./submit.sh ./prorun.sh ${fname}
+  ./submit.sh ./prorun.sh ${fname}
+
+  let njobs=$(condor_q | tail -3 | awk '/sphnxpro/ { print $4 }')
+  while [[ $njobs -gt 900 ]]
+  do
+    sleep 300
+    let njobs=$(condor_q | tail -3 | awk '/sphnxpro/ { print $4 }')
+  done
+done
```

Shellcheck (0.11.0)

[info] Command appears to be unreachable. Check usage (or ignore if invoked indirectly).

(SC2317)